doc/classes/@GlobalScope.xml

```diff
@@ -2143,49 +2143,49 @@
 			Space key.
 		</constant>
 		<constant name="KEY_EXCLAM" value="33" enum="Key">
-			! key.
+			Exclamation mark ([code]![/code]) key.
 		</constant>
 		<constant name="KEY_QUOTEDBL" value="34" enum="Key">
-			" key.
+			Double quotation mark ([code]"[/code]) key.
 		</constant>
-		<constant name="KEY_NUMBERSIGN" value="35" enum="Key">
-			# key.
+		<constant name="KEY_NUMBERSIGN" value="35" enum="Key" keywords="pound, hash">
+			Number sign or [i]hash[/i] ([code]#[/code]) key.
 		</constant>
 		<constant name="KEY_DOLLAR" value="36" enum="Key">
-			$ key.
+			Dollar sign ([code]$[/code]) key.
 		</constant>
 		<constant name="KEY_PERCENT" value="37" enum="Key">
-			% key.
+			Percent sign ([code]%[/code]) key.
 		</constant>
 		<constant name="KEY_AMPERSAND" value="38" enum="Key">
-			&amp; key.
+			Ampersand ([code]&amp;[/code]) key.
 		</constant>
 		<constant name="KEY_APOSTROPHE" value="39" enum="Key">
-			' key.
+			Apostrophe ([code]'[/code]) key.
 		</constant>
-		<constant name="KEY_PARENLEFT" value="40" enum="Key">
-			( key.
+		<constant name="KEY_PARENLEFT" value="40" enum="Key" keywords="round_bracket_left">
```

Comment:
I still don't understand why we're using "snake_case" keywords here?

Keywords are related words that users might type in the search bar. They're meant to be *natural language*, not pseudo API.

I would just put "round bracket left" as a keyword, or no keywords.

Reply:
Unsure now that I think about it, it's just been convention. Keywords don't really rely on the underscore

Reply:
Updated the PR. Because they're not in `snake_case` anymore, I believed it was worth rewriting the "left" and "right" bracket keywords to be "open" and "close", respectively.